W-16543932 Terminology and wording cleanup (#650)
mulesoft/docs-gateway: mule-gateway/modules/ROOT/pages/mule-gateway-encryption-mule4.adoc

--

[NOTE]

> The `![` prefix and  `]` suffix are required because they specify to Mule Runtime Engine that the enclosed value is encrypted.

Reviewer: ```suggestion
The required `![` prefix and  `]` suffix indicate to Mule runtime engine that the enclosed value is encrypted.
```

> When starting Mule runtime engine, you must pass your secret key, using the `anypoint.platform.encryption_key` property:

Reviewer: I wonder if we can sync on whether to use Mule, Mule runtime engine, or Mule runtime for subsequent mentions.

Author: I think we should say "Mule runtime" on subsequent mentions, just because we say "Mule blahblah" about some other things which are not Mule runtime engine, and so it's parallel and *maybe* easier for translation?

I did not follow that in this set of changes. I can make another pass tomorrow or let it be.
